README.md

```diff
@@ -145,6 +156,174 @@ readData(dataInput) {
 }
 ```
 
+# Setting Up Client Network
+
+### 7. Enabling Client TLS/SSL
+
+You can use TLS/SSL to secure the connection between the client and members. If you want TLS/SSL enabled
```

Comment:
If there is a possibility to have more than one client, let's use "clients" here.